src/libfetchers/git.cc

```diff
@@ -393,10 +393,10 @@ struct GitInputScheme : InputScheme
         {
             if (workdirInfo.isDirty) {
                 if (!settings.allowDirty)
-                    throw Error("Git tree '%s' is dirty", locationToArg());
+                    throw Error("Git tree '%s' is dirty with uncommitted changes", locationToArg());
```

Comment:
Why not just say "git tree has uncommitted changes" and drop "is dirty" entirely

Reply:
I waffled on that. The issue itself actually says drop "dirty" altogether, and then I put it back here because "dirty" is a searchable term in the context of git ... so that seemed useful? I dunno, what do you think about that?

Reply:
:+1: on "has uncommitted changes".

Reply:
Done!